command/format.go

```diff
@@ -214,20 +209,58 @@ func (p PrettyFormatter) OutputAutopilotState(ui cli.Ui, data interface{}) {
 		outputStringSlice(&buffer, "   ", state.NonVoters)
 	}
 
+	if state.OptimisticFailureTolerance > 0 {
+		buffer.WriteString(fmt.Sprintf("Optimistic Failure Tolerance:    %d\n", state.OptimisticFailureTolerance))
+	}
+
+	// Servers
 	buffer.WriteString("Servers:\n")
 	var outputs []mapOutput
 	for id, srv := range state.Servers {
 		outputs = append(outputs, mapOutput{key: id, value: formatServer(srv)})
 	}
-
 	sort.Slice(outputs, func(i, j int) bool {
 		return outputs[i].key < outputs[j].key
 	})
-
 	for _, output := range outputs {
 		buffer.WriteString(output.value)
 	}
 
+	// Redundancy Zones
+	if len(state.RedundancyZones) > 0 {
+		buffer.WriteString("Redundancy Zones:\n")
+		for zoneName, zone := range state.RedundancyZones {
+			buffer.WriteString(fmt.Sprintf("   %s\n", zoneName))
+			buffer.WriteString(fmt.Sprintf("      Servers: %s\n", strings.Join(zone.Servers, ", ")))
+			buffer.WriteString(fmt.Sprintf("      Voters: %s\n", strings.Join(zone.Voters, ", ")))
+			buffer.WriteString(fmt.Sprintf("      Failure Tolerance: %d\n", zone.FailureTolerance))
+		}
+	}
+
+	// Upgrade Info
+	if state.Upgrade != nil {
+		buffer.WriteString("Upgrade Info:\n")
+		buffer.WriteString(fmt.Sprintf("   Status: %s\n", state.Upgrade.Status))
+		buffer.WriteString(fmt.Sprintf("   Target Version: %s\n", state.Upgrade.TargetVersion))
+		buffer.WriteString(fmt.Sprintf("   Target Version Voters: %s\n", strings.Join(state.Upgrade.TargetVersionVoters, ", ")))
+		buffer.WriteString(fmt.Sprintf("   Target Version Non-Voters: %s\n", strings.Join(state.Upgrade.TargetVersionNonVoters, ", ")))
+		buffer.WriteString(fmt.Sprintf("   Target Version Read Replicas: %s\n", strings.Join(state.Upgrade.TargetVersionReadReplicas, ", "))) // TODO: ok to use this term? how is this different from non-voters?
```

Comment:
Read replica is the consul equivalent of non-voter.  We don't like that term on the vault side because it's often used for perf secondaries too - which I know doesn't make sense, but regardless it makes the term ambiguous.

Reply:
I'll remove this.